qutebrowser/config/configdata.yml

```diff
@@ -1247,9 +1247,14 @@ tabs.favicons.scale:
     `tabs.padding`.
 
 tabs.favicons.show:
-  default: true
-  type: Bool
-  desc: Show favicons in the tab bar.
+  default: always
```

Comment:
This needs to have a config migration, but I'm not sure how that's handled. Just a pointer for later.

I'm not 100% sure about this, but a similar problem was solved in https://github.com/qutebrowser/qutebrowser/pull/3577 as well

Reply:
You'd need to do something like this:

https://github.com/qutebrowser/qutebrowser/blob/8090d3e28940f02e94858d80293dcfcd8a635fd6/qutebrowser/config/configfiles.py#L251-L263